plugins/destination/clickhouse/queries/testdata/create_table.sql

```diff
@@ -1,9 +1,9 @@
 CREATE TABLE `table_name` (
-  `_cq_id` UUID,
-  `_cq_parent_id` Nullable(UUID),
-  `_cq_source_name` Nullable(String),
-  `_cq_sync_time` Nullable(DateTime64(9)),
-  `extra_col` Float64,
-  `extra_inet_col` Nullable(String),
-  `extra_inet_arr_col` Array(Nullable(String))
-) ENGINE = MergeTree ORDER BY (`_cq_id`)
\ No newline at end of file
+`_cq_id` UUID,
+`_cq_parent_id` Nullable(UUID),
+`_cq_source_name` Nullable(String),
+`_cq_sync_time` Nullable(DateTime64(9)),
+`extra_col` Float64,
+`extra_inet_col` Nullable(String),
+`extra_inet_arr_col` Array(Nullable(String)),
```

Comment:
wrong, CH should fail on extra coma

Reply:
It doesn't (see the testing passing) and also https://clickhouse.com/docs/en/sql-reference/statements/create/table/

![image](https://user-images.githubusercontent.com/26760571/220115974-340296c4-cfa6-4e8c-ac92-e7518d4223f9.png)